src/components/Keyword/Keywords.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useRef } from 'react'
+import { CheckSquareFilled } from '@ant-design/icons'
+import { KeywordGroup, KeywordList, KeywordName } from './Keywords.styles'
+
+interface KeywordProps {
+  name: string
+  array: string[]
+}
+
+const Keywords: React.FC<KeywordProps> = ({ name, array }) => {
+  const listRef = useRef<HTMLLIElement[] | null[]>([])
```

Comment:
(p4) null 의 배열..인가요? 이런것도 가능하군요 map 을 이용하는건 어떨까 합니다

```suggestion
useRef<{ [idx: number]: HTMLLIElement >({})
```